### DISCUSSION TITLE
Support of errsole UI on winston logs

### DISCUSSION THREAD

**Discussion by baidwan007** [errsole/errsole.js#95] [Ideas]
If errsoles UI could be made available to look into winston logs and search through them then that would be a great feature for people who are already using winston. You already have a good UI and letting people use it for their existing winston logs is a win win for everyone.

**Comment by mrrishimeena** [comment]
@baidwan007,

We’ve added a Winston transport, so any developer already using Winston in their current project can still take advantage of Errsole. You can find more details here: [https://github.com/errsole/errsole.js/blob/master/docs/winston-errsole.md](https://github.com/errsole/errsole.js/blob/master/docs/winston-errsole.md)
